girder/utility/assetstore_utilities.py

```diff
@@ -60,6 +59,15 @@ def getAssetstoreAdapter(assetstore, instance=True):
         return cls
 
 
+def setAssetstoreAdapter(storeType, cls):
+    global assetstoreTable
+
+    if storeType not in assetstoreTable:
+        raise Exception('Illegal assetstore type code: "%s"' % (storeType))
```

Comment:
It would be good if registering a listener for the `assetstore.adapter.get` event printed a deprecation warning log message (as we have in some other places).